src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
-    @Override
-    public void put(K key, V value) {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private int threshold;
+    private int capacity;
```

Comment:
I think we could live without this variable, with using just table.length

Reply:
I use it in the resize () method so that the hash () method works correctly.